src/server/systems/NetworkingServer.cpp

```diff
@@ -194,9 +218,38 @@ void NetworkingServer::setClientUnreliableUpdate(sf::Packet packet, int client_i
                 m_playerUnreliableUpdates_lock);
             m_playerUnreliableUpdates_t.push_back(player_unreliable_input);
         }
-    } else {
-        std::cout << "Receive client_update command with tick drifted past drift threshold. "
-                  << "Drift value is: " << drift << std::endl;
+    }
+}
+
+
+float NetworkingServer::getBroadcastGameStateRate() {
+    {
+        std::lock_guard<std::mutex> m_playerIdsToUpdatesMetric_guard(m_playerIdsToUpdatesMetric_lock);
+        return m_broadcastGameStateMetric_t.getRate(sf::seconds(1));
+    }
+}
+
+std::unordered_map<sf::Uint32, float> NetworkingServer::getPlayerUnreliableUpdatesRates() {
+    {
+        std::lock_guard<std::mutex> m_playerIdsToUpdatesMetric_guard(m_playerIdsToUpdatesMetric_lock);
+        auto map = m_playerIdsToUpdatesMetric_t;
+        for (auto it = map.begin(); it != map.end(); ++it) {
```

Comment:
can you change these iterator loops to the range based style, it's much more readable 

```
for (const auto& kv : myMap) {
    std::cout << kv.first << " has value " << kv.second << std::endl;
}
```